### PR TITLE
Make IsWindowsService() visible publicly

### PR DESCRIPTION
**Pull request by drmcclelland**
Make IsWindowsService() visible publicly by moving it a new Microsoft.Extensions.Hosting.WindowsServices.ServiceHelpers type.

Addresses #1337

### CODE REVIEW

**Comment by Tratcher** [comment]
Housekeeping, you need to update the ref assemblies to add the new public API:
`Generated code is not up to date. You might need to regenerate the reference assemblies or project list (see docs/ReferenceAssemblies.md and docs/ReferenceResolution.md)`

File: src/Hosting/WindowsServices/src/ServiceHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.Hosting.WindowsServices
+{
+    public class ServiceHelpers
```
Comment (campersau) [inline src/Hosting/WindowsServices/src/ServiceHelpers.cs]: Should this be a `static` class?

Comment (drmcclelland) [reply src/Hosting/WindowsServices/src/ServiceHelpers.cs]: Thanks, updated code in PR.

**Comment by Tratcher** [comment]
Thanks

**Pull request merged**